grails-app/services/au/org/ala/collectory/SitemapService.groovy

```diff
@@ -40,6 +44,8 @@ class SitemapService {
     // run daily, initial delay 1hr
     @Scheduled(fixedDelay = 86400000L, initialDelay = 3600000L)
     def build() throws Exception {
+        fileCount = 0;
```

Comment:
`fileCount` needs to be reset each time the method is run to prevent mismatch between entries in the sitemap index file and the actual sitemap files created